packages/web/src/pages/chat-page/components/ChatBlastHeader.tsx

```diff
@@ -57,7 +58,7 @@ export const ChatBlastHeader = ({ chat }: { chat: ChatBlast }) => {
         <Flex gap='s' alignItems='center'>
           <IconUserList size='m' color='default' />
           <Text variant='title' size='xl'>
-            {audienceCount}
+            {formatCount(audienceCount ?? 0)}
```

Comment:
how come we need `?? 0` here but not in the other places? should we just not show the audience count until it loads in?